CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 *These will be removed in the next major release*
 - N/A
 
+1.0.1
```

Comment:
Need to update the version in the changelog

Reply:
Oops. Updated.